students/chaycasso/Lesson06/mailroom4.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+#
+# Assignment: Mailroom, Part 4
+# Chay Casso
+# 2/24/2018
+
+from collections import OrderedDict
+
+class Mailroom(object):
+
+    def __init__(self):
+    # Initial donor table with the donation values.
+        self.donor_table_dict = {"William Gates, III": [401321.52, 201342.71], "Mark Zuckerberg": [123.45, 5123.21, 8213.11],
+                        "Jeff Bezos": [877.33], "Paul Allen": [152.42, 30.54, 825.21], "Steve Ballmer": [5198.96, 654.98]}
+        answer = ""
+
+
+    def thank_you(self, thank_you_dict, full_name, donation_value_str):
```

Comment:
cast `full_name` to lower when you pass it to this function so you don't have to lower it on every instance.